mirna_explorer/repository/create_database.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import os
+import sqlite3
+
+data_folder = "../../../mirna_target_prediction/data/"
+
+required_files = ["raw/UTR_Sequences.txt",
+				  "processed/mirna_id_dictionary.tsv",
+				  "processed/gene_id_dictionary.tsv",
+				  "processed/scored_interactions_processed.tsv"]
+
+for filename in required_files:
+	path = data_folder + filename
+	if not os.path.isfile(path):
+		print(f'error: file not found: \'{path}\', aborting')
+		exit(1)
+
+db_path = 'database/db.sqlite'
+
+if os.path.isfile(db_path):
+	print(f'the database is already present at {db_path}, aborting')
+	exit(1)
+		
+db_connection = sqlite3.connect('database/db.sqlite')
+db = db_connection.cursor()
+
+with open('./model/design.sql', 'r') as infile:
```

Comment:
`'r'` is the default mode for `open()`. Therefore, you can omit the parameter, i.e. `with open('./model/design.sql') as infile`